Source/WebCore/PAL/pal/PALSwift/UnsafeOverlays.swift

```diff
@@ -37,5 +37,67 @@ extension HashFunction {
                 start: data.__dataUnsafe(), count: data.size()))
     }
 }
+extension CryptoKit.Digest {
+    public func toVectorUInt8<T: HashFunction>(_: T.Type) -> VectorUInt8 {
+        var rv = VectorUInt8(T.Digest.byteCount)
+        var index = 0
+        self.forEach { val in
+            rv[index] = val
+            index += 1
+        }
+        return rv
+    }
```

Comment:
The thesis of UnsafeOverlays is:
* The implementation is allowed to perform unsafe operations
* The output should be safe objects that enforce safety invariants

Consistent with that thesis, we can do something more efficient here than byte-by-byte copy:

```
return self.withUnsafeBytes { unsafeBytes in
    return WTFMove(Vector(span(unsafeBytes.baseAddress!, unsafeBytes.count)
}
```